### PR TITLE
DS_Week2_복습과제_차수빈

### PR DESCRIPTION
**Pull request by chasubeen**

### CODE REVIEW

**Comment by kkyung0131** [comment]
1.3
열의 이름을 알고 싶을 때 
데이터명.columns로 확인할 수 있습니다!(ex. train_df.columns)

1.6 
더 간단하게 인코딩하신 것 같아요!
레이블 인코딩을 할 수 있는 다른 코드도 보시면 좋을 것 같아 적어드립니다!
1)(파머완 코드)
items = ['white','red']
encoder = LabelEncoder()
encoder.fit(items)
labels = encoder.transform(items)
train_df['type'] = encoder.fit_transform(train_df['type'].values)
print(train_df['type'].value_counts())

2)(여채윤벗, 여유진벗, 소예림벗, 황선경벗 코드)
encoder= LabelEncoder()
train_df['type'] = encoder.fit_transform(train_df['type'])

3)(조혜빈벗 코드)
items=['type']
le=LabelEncoder()
for i in items:
traindrop[i]=le.fit_transform(traindrop[i])
traindrop['type'].value_counts()  

2.1~2.6
우선 2번 문제 푸실 때, 데이터를 wine 데이터가 아니라 다른 데이터로 하셔서 전체적으로 값이 다른 것 같아요! 그래서 코드 위주로 살펴봤는데, 딱히 문제있는 코드는 안 보여요! (2.6의 경우 wine 데이터와는 다르게 변수의 개수가 많기 때문에 top 20으로 지정해주셨어도 상관없을 것 같습니다!)

3.2 
문제에 제시되어 있지 않지만 GridSearchCV 인자로 cv 값도 설정해주셨으면 더 좋은 코드가 되었을 것 같아요!

3.3 
best_estimator_를 이용하는 방법 말고 다른 방법도 있어서 적어드립니다.
lin_svm = SVC(kernel='linear',C=1)
lin_svm.fit(X_train,y_train)
pred = lin_svm.predict(X_test)
accuracy = accuracy_score(y_test, pred)
print('accuracy : {0:.4f}'.format(accuracy))
